jiant/tasks/evaluate/core.py

```diff
@@ -111,6 +111,30 @@ def compute_metrics_from_preds_and_labels(cls, preds, labels):
         return Metrics(major=acc, minor={"acc": acc})
 
 
+class MultiLabelAccAndF1EvaluationScheme(BaseLogitsEvaluationScheme):
+    def get_labels_from_cache_and_examples(self, task, cache, examples):
+        return get_multi_label_ids_from_cache(cache=cache)
+
+    def get_preds_from_accumulator(self, task, accumulator):
+        logits = accumulator.get_accumulated()
+        return (logits > 0.5).astype(int)
+
+    @classmethod
+    def compute_metrics_from_preds_and_labels(cls, preds, labels):
+        # noinspection PyUnresolvedReferences
+        acc = float((preds == labels).mean())
+        labels = np.array(labels)
+        minor = {
+            "acc": acc,
+            "f1_micro": f1_score(y_true=labels, y_pred=preds, average="micro"),
```

Comment:
There was some confusion recently about which F1 measure was appropriate. I threw them all in for testing. These will be trimmed before merging this PR.

Reply:
Are these still being trimmed?

Reply:
Thanks for the reminder, trimmed w/ fb5f135.